docs/docs/guides/rego-policies/rego-policies.mdx

```diff
@@ -91,6 +91,54 @@ policies:
 ```
 Check our [policies reference](/reference/policies) for more information on how to attach policies to contracts.
 
+## Policy inputs
```

Comment:
should we also indicate how to add inputs to the spec itself? Do we have anywhere how the spec looks like?

Reply:
Yes, in the policies reference doc: https://docs.chainloop.dev/reference/policies#policy-arguments